Examples/test-suite/typescript_import.i

```diff
@@ -19,3 +13,7 @@ class C {
 };
 %}
 
+%pragma(javascript) tsimports=%{
+import { D } from './typescript-import-d-types';
+%}
+
```

Comment:
It doesn't really matter, of course, but I'd keep this pragma above, it makes more sense to order things in the same order in the `.i` file in which they appear in the output, IMHO.

Reply:
No. It is not that difficult to do. I need to collect all the pieces into a collection of interfaces/enums and one string for the pragma and in the end write them. Or maybe do it with two string (one for the interfaces/enums and another one for the pragma) to keep it simple.

Reply:
Yes, I'd just use 2 strings (and add more if more are needed later, of course).